docs/platform/what-is-tizen/versions/tizen-5-5-m2.md

```diff
@@ -0,0 +1,49 @@
+
+- **Tizen 5.5’s Watchface Complication APIs** provide easier way to create custom complication in watchface. APIs provide various kinds of complication types. Developers can provide various kinds of complication services such as health, battery, air pollution etc. by using APIs.
+
+- **Tizen 5.5 supports Lottie animation**
+ Lottie which was named by AirBnB, is library that renders After Effects animations in real time. The lottie file is composed by using JSON format, it contains visual shape information such as rectangle, circle, path and fill methods like the traditional vector graphics elements.
+- **Multi-assistant Framework**
+Aims to support various AI assistants on the same time for giving a better usability. The framework defines a common interface for making an assistant of multi-assistant framework. And it provides an expandable wake-up engine that recognizes invocation keyword for each assistant and supports to add a new keyword.
+
+- **The multi-device distributed web engine** is designed for both memory resource requirements mitigation and JavaScript execution speed enhancement to overcome the resource constraints of the low-end devices. With the prior multi-process design of the web engine, the memory usage of the renderers becomes significant as multiple tabs are created. CPU resource consumption of the renderer processes is also substantial as web applications get complicated. To manage hardware resources and to make resource usage predictable in low-end devices, the multi-device distributed web engine introduces features for offloading renderer processes over the network to resource-free devices while the browser process runs on the local low-end device.
+
+- **User Awareness Framework**
+Smart devices could provide useful services based on the user presence context. Tizen 5.5 supports User Awareness Framework based on multiple sensors. Currently, Wi-Fi, BLE, motion, and light sensors are supported and it could be extended to support other types of sensors based plug-in architecture.
+
+
+## Intelligent/AI/On Device AI
+
+- **A new Tizen domain, Machine Learning (ML)**, is introduced along with its ML Inference API sets: “Pipeline ML API” and “Single ML API”. ML Inference API sets allows application developers to apply their machine learning mechanisms or neural network models easily and efficiently. Especially if the intelligence application has multiple sensors, multiple neural network models, and complex topology of data paths, Pipeline ML API can help implement the system efficiently with less time and effort. If developers need to use neural network models preloaded with the platform, APIs in Media Vision are appropriate because ML Inference APIs require developers to supply neural network models. In other words, for predefined special functions, Media Vision is appropriate and for general or custom functions, ML Inference is appropriate.
```

Comment:
@mijin85cho  applied your comments, thanks ! 

- **A new Tizen domain, Machine Learning (ML):** ML is introduced along with Inference API sets: Pipeline ML API and Single ML API. ML Inference API sets allow you to apply your machine learning mechanisms or neural network models easily and efficiently. For example, if your intelligent application has multiple sensors, multiple neural network models, and complex topology of data paths. 
The Pipeline ML API can help you to implement the system efficiently with less time and effort. If you need to use the neural network models preloaded with the platform,  Media Vision APIs are appropriate because ML Inference APIs require you to supply neural network models. In other words, for predefined special functions, Media Vision is appropriate to use and for general or custom functions, ML Inference is appropriate to use.